DOCSP 33265 Resilience of mongosync (#201)
mongodb/docs-cluster-to-cluster-sync: source/reference/mongosync.txt

.. include:: /includes/collections/behavior-system-collections.rst

Resilience
~~~~~~~~~~

> ``mongosync`` is resilient and able to handle non-fatal errors. Error
> and failure log entries do not indicate that ``mongosync`` is failing or

Reviewer: ```suggestion
``mongosync`` is resilient and able to handle non-fatal errors. Logs that
contain the word "error" or "failure" do not indicate that ``mongosync`` is failing or
```

Author: Done!

.. include:: /includes/collections/behavior-system-collections.rst

Resilience
~~~~~~~~~~

``mongosync`` is resilient and able to handle non-fatal errors.

> ``error`` and ``failure`` log entries do not indicate that ``mongosync``

Reviewer 2: ```suggestion
Error and failure log entries do not indicate that ``mongosync``
```

is failing or corrupting data. For example, if a network error occurs,

> ``mongosync`` writes an error log but still completes sync. In the

Reviewer 2: ```suggestion
``mongosync`` writes an error log but still completes the sync. In the
```

Author: Done!

> error occurs, ``mongosync`` writes an error log but still completes the

Reviewer: ```suggestion
error occurs, the ``mongosync`` logs may contain the word error, but mongosync is still able to complete the
```

Author: Done!

> ``mongosync`` writes an ``error`` log but still completes sync. In the

Reviewer 2: ```suggestion
``mongosync`` writes an error log but still completes sync. In the
```

> case that sync does not complete, ``mongosync`` writes a fatal log

Reviewer 2: ```suggestion
case that a sync does not complete, ``mongosync`` writes a fatal log
```

Author: Done! Thanks for your help on this one!

> case that sync does not complete, ``mongosync`` writes a ``fatal`` log

Reviewer 2: ```suggestion
case that sync does not complete, ``mongosync`` writes a fatal log
```

> fatal log entry. 

Reviewer: either is fine
